docs/source/changelog.rst

```diff
@@ -217,4 +219,4 @@ Other Changes
 #. Cleanup 3b1b Specific Files
 #. Rename package from manimlib to manim
 #. Move all imports to :code:`__init__`, so :code:`from manim import *` replaces :code:`from manimlib.imports import *`
-#. Global dir variable handling has been removed. Instead :code:`initialize_directories`, if needed, overrides the values from the cfg files at runtime.
\ No newline at end of file
+#. Global dir variable handling has been removed. Instead :code:`initialize_directories`, if needed, overrides the values from the cfg files at runtime.
```

Comment:
Can you remove changes in this file instead use the wiki.

Reply:
Oh, I gladly will. We should change the info when one opens a new PR.

Reply:
That's in new template.

Reply:
Good. Actually, I just realized that I do not seem have write permissions to the wiki.

Reply:
That's okay; if you provide a one-line summary of your PR (which is now also a new section in the new PR template), I'll include it in the wiki.

Reply:
I'd add: 

> Added a class :class:`MarkupText` (via :pr:`855`) for easier and more flexible text formatting

Will also include this into the PR text